ui/src/components/Aside/index.vue

```diff
@@ -248,7 +248,9 @@ function searchGroup() {
     let groupLists = []
     data.groupListsCopy.forEach(item => {
         let itemCache = JSON.parse(JSON.stringify(item))
-        if (Array.isArray(itemCache.children)) {
+        if (itemCache.metadata.name.indexOf(data.search) > -1) {
+            groupLists.push(itemCache)
```

Comment:
It's better to display any matching group or item. Currently, the solution skips matched terms if the group is matched.

Reply:
Could be more specific? I might not get your point.

Reply:
If any items under the group match the keyword, it should also be shown.

groupA:
  - itemA
  - itemB

When you issue "A", only groupA and itemA should show up instead of all of them.